commons/monitor/questionnaire/questionnaire_application_interaction_event.avsc

```diff
@@ -0,0 +1,19 @@
+{
+  "namespace": "org.radarcns.monitor.questionnaire",
+  "type": "record",
+  "name": "QuestionnaireApplicationInteractionEvent",
+  "doc": "Schema of reporting events for user interaction with the questionnaire application. This describes significant user interactions, such as response to notifications and starting or completing questionnaires.",
+  "fields": [
+    { "name": "time", "type": "double", "doc": "Device timestamp in UTC (s)." },
+    { "name": "eventType", "type": {
+      "name": "InteractionEventType",
+      "doc": "NOTIFICATION_OPEN = When a user presses the notification to open the app, APP_OPEN = When a user opens the app directly, QUESTIONNAIRE_STARTED = When a user starts a questionnaire, QUESTIONNAIRE_FINISHED = When a user completes a questionnaire, QUESTIONNAIRE_CLOSED = When a user closes a questionnaire without completing it.", 
+      "type": "enum",
+      "symbols": ["NOTIFICATION_OPEN", "APP_OPEN", "QUESTIONNAIRE_STARTED", "QUESTIONNAIRE_FINISHED", "QUESTIONNAIRE_CLOSED", "OTHER", "UNKNOWN"],
```

Comment:
Just for clarity, would it make sense to rename `QUESTIONNAIRE_FINISHED` to `QUESTIONNAIRE_COMPLETED` and `QUESTIONNAIRE_CLOSED` to `QUESTIONNAIRE_CANCELLED`?

Reply:
This line should not end with a comma

Reply:
Maybe we can keep `QUESTIONNAIRE_FINISHED` because this is already what's on Firebase analytics? But I can change `QUESTIONNAIRE_CLOSED` since it is not yet on Firebase.